coral/docs/development-with-local-klaw.md

```diff
@@ -21,20 +24,33 @@ Please check out the [proxy README](../proxy/README.md) for more detailed inform
   -> see [nvmrc](../.nvmrc) or the `engines` definition in [package.json](../package.json) for version).
 - Coral uses [pnpm](https://pnpm.io/) (version 7) as a package manager. Read their official documentation [how to install](https://pnpm.io/installation) pnpm.
 
-1. navigate to [`/coral`](../../coral)
-2. run `pnpm install`
-3. run `pnpm add-precommit` the first time you install the repository to set the custom directory for our pre commit hooks.
-4. go to directory [`coral/proxy`](../../coral/proxy)
-5. run `pnpm install` there, too
-6. Run development:
-   6.1. If you have not setup the docker environments, please follow [First setup]()
-   6.2. If you already have a setup, run `pnmv dev:[start|restart]`
+1. Navigate to [`/coral`](../../coral)
+2. Run `pnpm install`
+3. Run `pnpm add-precommit` the first time you install the repository to set the custom directory for our pre commit hooks.
+4. Go to directory [`coral/proxy`](../../coral/proxy)
+5. Run `pnpm install` there, too
+6. Install Docker - [Get Docker](https://docs.docker.com/get-docker/)
+7. Go to to directory [`coral/proxy`](../../coral/proxy)
+8. Run `pnpm install` if you haven't already
+9. Run `ppm setup`
+   `pnpm setup` will build and deploy your docker container for Klaw core, cluster api and a test environment for klaw (zookeeper, kafka, schema-registry).
+10. Run `pnmv dev:[start|restart]`
     - check out the [documentation](../../coral/proxy/README.md) for more scripts and information when to use them.
     - the proxy runs on [`http://localhost:1337`](http://localhost:1337)
-  - ❗️ **The correct redirect for login and authentication is **not** working yet.** To authenticate yourself see [ Login and authentication]
-    
+    - Login - ❗️The correct redirect for login and authentication is **not** working in the proxy yet. To login:
```

Comment:
Is this missing a line break for this bullet point?
```
- ❗️The correct redirect for login and authentication is **not** working in the proxy yet. To login:
```

Reply:
It was supposed to be like an "insertion" (sorry I can't find the right translation for this), but I think for that you don't use blanks with the dash 🤔   

Like, it belongs to the "Login" and is meant as an additional info. I put it in () now, I hope that this is more clear.